plugin/src/main/java/org/godotengine/openxr/vendors/GodotPanelApp.kt

```diff
@@ -61,10 +60,9 @@ class GodotPanelApp : GodotActivity() {
 		super.onCreate(savedInstanceState)
 	}
 
-	@CallSuper
-	protected override fun updateCommandLineParams(args: Array<String>) {
-		// Force XR to be turned off and discard other params.
-		super.updateCommandLineParams(arrayOf("--xr_mode_regular", "--xr-mode", "off"))
+	override fun getCommandLine(): MutableList<String> {
+		// Force XR to be turned off.
+		return mutableListOf("--xr_mode_regular", "--xr-mode", "off")
```

Comment:
We'd want to append to the parent command line arguments to ensure we don't inadvertently break some features. If there is conflicts with the arguments we're adding, we should remove the conflicting arguments and keep the rest.

Reply:
Updated in my latest push